static/app/components/searchQueryBuilder/tokens/useSortedFilterKeyItems.tsx

```diff
@@ -216,18 +221,28 @@ export function useSortedFilterKeyItems({
         (!keyItems.length || inputValue.trim().includes(' ')) &&
         (!replaceRawSearchKeys?.length || !hasRawSearchReplacement);
 
+      const options: Array<RawSearchFilterIsValueItem | RawSearchFilterHasValueItem> =
+        replaceRawSearchKeys?.map(key => {
+          const value = inputValue?.includes(' ')
+            ? `"${inputValue.replace(/"/g, '')}"`
+            : inputValue;
+
+          return createRawSearchFilterIsValueItem(key, value);
+        }) ?? [];
+
+      if (hasWildcardSearch) {
```

Comment:
IMO this should be the first option when available - it's probably what people want most frequently